docs/docs/migration/migration-from-v5-to-v6.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Migrating from v5 to v6
+sidebar_position: 1
+---
+
+# Migrating from v5 to v6
+
+First, execute `npx @comet/upgrade v6` in the root of your project. 
```

Comment:
```suggestion
First, execute `npx @comet/upgrade@latest v6` in the root of your project. 
```

We probably need the latest tag due to npx caching

Reply:
Done